addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -20,6 +20,16 @@ function factory(dependencies) {
         // Public
         //----------------------------------------------------------------------
 
+        /**
+         * @returns {boolean}
+         */
+        isNotificationPermissionDefault() {
+            const windowNotification = this.env.browser.Notification;
+            return windowNotification
+                ? windowNotification.permission === 'default'
+                : false;
+        }
```

Comment:
Could be turned into a computed field... just unfortunate need to manually re-compute because `this.env.browser` doesn't sync with model fields... Could be left like this for the moment.

Reply:
Yep, needs to be recomputed every time specifically. Can't be a compute if we can't express dependencies.